client/ayon_core/plugins/publish/integrate_attach_reviewable.py

```diff
@@ -0,0 +1,136 @@
+        # TODO: Reviewable frame range and resolutions
+        #  Because we are attaching the data to another instance, how do we
+        #  correctly propagate the resolution + frame rate to the other
+        #  instance? Do we even need to?
+        # TODO: If this were to attach 'renders' to another instance that would
+        #  mean there wouldn't necessarily be a render publish separate as a
+        #  result. Is that correct expected behavior?
+        attr_values = self.get_attr_values_from_data(instance.data)
+        attach_to = attr_values.get("attach", [])
+        if not attach_to:
+            self.log.debug(
+                "Reviewable is not set to attach to another instance."
+            )
+            return
+
+        attach_instances: List[pyblish.api.Instance] = []
+        for attach_instance_id in attach_to:
+            # Find the `pyblish.api.Instance` matching the `CreatedInstance.id`
+            # in the `attach_to` list
+            attach_instance = next(
+                (
+                    _inst
+                    for _inst in instance.context
+                    if _inst.data.get("instance_id") == attach_instance_id
+                ),
+                None,
+            )
+            if attach_instance is None:
+                continue
+
+            # Skip inactive instances
+            if not attach_instance.data.get("active", True):
+                continue
+
+            # For now do not support attaching to 'farm' instances until we
+            # can pass the 'attaching' on to the farm jobs.
+            if attach_instance.data.get("farm"):
+                self.log.warning(
+                    "Attaching to farm instances is not supported yet."
+                )
+                continue
+
+            attach_instances.append(attach_instance)
+
+        instances_names = ", ".join(
+            instance.name for instance in attach_instances
+        )
+        self.log.info(
+            f"Attaching reviewable to other instances: {instances_names}"
+        )
+
+        # Copy the representations of this reviewable instance to the other
+        # instance
+        representations = instance.data.get("representations", [])
+        for attach_instance in attach_instances:
+            self.log.info(f"Attaching to {attach_instance.name}")
+            attach_instance.data.setdefault("representations", []).extend(
+                copy.deepcopy(representations)
+            )
+
+        # Delete representations on the reviewable instance itself
+        for repre in representations:
+            self.log.debug(
+                "Marking representation as deleted because it was "
+                f"attached to other instances instead: {repre}"
+            )
+            repre.setdefault("tags", []).append("delete")
+
+        # Stop integrator from trying to integrate this instance
+        if attach_to:
+            instance.data["integrate"] = False
```

Comment:
Is this good idea for `render` instance?

Reply:
Yes - because what it integrates - then occurs on the _other_ instances. Basically representations get moved there instead.

Reply:
There is a `render` product type that is used as "preview render"?

Reply:
A turntable render in a lookdev scene for example, may be considered just a reviewable for the published look. So in that sense, I suppose so? You wouldn't do a viewport preview for that - you'd use a render.

Reply:
And isn't that confusing? Because it is from my point of view, I have an instance that is suddenly not integrated because some other instance has it set as review source, the instance itself can't affect it at all, and has no visual information about it. And it might makes sense in Maya, but it doesn't make sense in e.g. Nuke or Fusion (and other 2D dccs), or?

Reply:
> And isn't that confusing? Because it is from my point of view, I have an instance that is suddenly not integrated because some other instance has it set as review source, the instance itself can't affect it at all, and has no visual information about it.

It is exactly the intention here. Of course it can be abused, but I'd argue that's not for us to police.

Reply:
Thanks, resolving this conversation.